### DISCUSSION TITLE
本项目 GitHub Pages 已发布 | The GitHub Pages has been released

### DISCUSSION THREAD

**Discussion by yanglbme** [doocs/advanced-java#1] [Announcements]
## 在线阅读
为了大家有一个更好的阅读体验，此项目已发布在 [GitHub Pages](https://doocs.github.io/advanced-java) 上。使用国内网络的朋友也可以访问 [Gitee Pages](https://doocs.gitee.io/advanced-java)，加载速度更快。
![image](https://user-images.githubusercontent.com/21008209/56121133-1600c280-5fa2-11e9-851c-204b59dad8b9.png)

![image](https://user-images.githubusercontent.com/21008209/56121091-01bcc580-5fa2-11e9-99a2-e7047e329e71.png)

## 项目动态
欢迎 **star** 关注本项目，你的关注会激励我一直坚持抽空补充、完善、修正项目内容；也欢迎在 [Issues](https://github.com/doocs/advanced-java/issues/) 讨论区交流，交流互动可以让这个项目变得更好。

## 转载说明
目前网上出现很多直接拷贝他人文章而不注明出处的，其实是完全无视他人劳动成果。如果你有转载本仓库文章的需求，请微信联系 YLB0109 授权转载，禁止私自转载。

**Comment by CraKeyBoy** [comment]
刚看了下 chrome 浏览器 cpu 使用率一直维持在 6%~10% 之间，静态页面也有运算？
![image](https://user-images.githubusercontent.com/1518543/50276678-7c0a8c80-047d-11e9-87ac-6f2a77410897.png)

**Comment by yanglbme** [comment]
@javmain ~~页面引入了一些 JS 插件库，可能耗了些性能~~

**Comment by yanglbme** [comment]
@javmain 我详细阅读了 docsify 的官方文档，它不会生成静态 html 文件，而是在运行时对 markdown 文件做的转换，可以看一下[这个说明](https://docsify.js.org/#/?id=what-it-is)。

**Comment by CraKeyBoy** [comment]
嗯，明白了。这个域名下的资源翻墙才能看到 unpkg.com，不知道其他人有这个问题没。 @yanglbme

**Comment by CraKeyBoy** [comment]
看来下，果然是这个
![image](https://user-images.githubusercontent.com/1518543/50321134-4e1f5980-050a-11e9-89e0-3fc7921ac8ed.png)

**Comment by yanglbme** [comment]
@javmain 😁 对的~

**Comment by NGLSL** [comment]
看起来很不错，已经star

**Comment by yanglbme** [comment]
@NGLSL thanks

**Comment by Jancd** [comment]
GitHub Page 浏览出了点问题，内容看不全。比如这个链接：https://doocs.github.io/advanced-java/#/docs/high-concurrency/why-mq?id=%E4%B8%BA%E4%BB%80%E4%B9%88%E4%BD%BF%E7%94%A8%E6%B6%88%E6%81%AF%E9%98%9F%E5%88%97（win7 Chrome）

**Comment by yanglbme** [comment]
@7Ethan 谢谢反馈，我刚刚看了，没有问题。你可以 refresh 刷新一下，有时候因为浏览器缓存，看到的不是最新的内容。

**Comment by Jancd** [comment]
> @7Ethan 谢谢反馈，我刚刚看了，没有问题。你可以 refresh 刷新一下，有时候因为浏览器缓存，看到的不是最新的内容。

换个浏览器没问题了。Chrome 页面卡死。

**Comment by yanglbme** [comment]
@7Ethan 好的

**Comment by ZhuPeng** [comment]
Hi, 最近开发了一个 GitHub 的微信小程序，可以方便的在手机上查看 GitHub 的相关内容，扫描下方二维码就可以方便查看本项目~
![image](https://7465-test-3c9b5e-1258459492.tcb.qcloud.la/qrcode.jpg)


小程序是开源的，有任何问题和意见可以直接提 Issue 反馈：[GitHub Trending Hub](https://github.com/ZhuPeng/mp-githubtrending)

**Comment by yanglbme** [comment]
@ZhuPeng 挺不错的 GitHub Trending 小程序。

欢迎其它朋友们使用该小程序阅读或实时查看 GitHub Trending 相关动态。

**Comment by skyformat99** [comment]
谢谢大牛牛啊
